src/photos.cs

```diff
@@ -944,9 +994,13 @@ interface PHImageManager {
 		CGSize MaximumSize { get; }
 
 		[iOS (9,1)]
-		[NoMac]
+		[Mac (10,15, onlyOn64: true)]
 		[Export ("requestLivePhotoForAsset:targetSize:contentMode:options:resultHandler:")]
 		int /* PHImageRequestID = int32_t */ RequestLivePhoto (PHAsset asset, CGSize targetSize, PHImageContentMode contentMode, [NullAllowed] PHLivePhotoRequestOptions options, PHImageManagerRequestLivePhoto resultHandler);
+
+		[TV (13,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Export ("requestImageDataAndOrientationForAsset:options:resultHandler:")]
+		int RequestImageDataAndOrientation (PHAsset asset, [NullAllowed] PHImageRequestOptions options, PHImageManagerRequestImageDataHandler resultHandler);
```

Comment:
Async?

Reply:
The documentation says that it is only called once if I am reading it correctly, so therefore it doesn't need an [async] correct? Here's what it says "@param resultHandler A block that is called exactly once either synchronously on the current thread or asynchronously on the main thread depending on the synchronous option specified in the PHImageRequestOptions options parameter (deliveryMode is ignored). Orientation is an EXIF orientation as an CGImagePropertyOrientation. For iOS or tvOS, convert this to an UIImageOrientation."

Reply:
The async will allow to have the async/await pattern which is nicer. Even if the cb is only called once, it will mean that we do not have to bloc the current thread.

Reply:
I've talk with @dalexsoto about this. While parse we could make the metho async, that long explanation make is scary and also means that the mono async/await might be doing evil things not expected by objc (for example, ConfigureAwait (false) + the wrong option).

Better leave it like this and later add the API if needed.

Reply:
Okay cool thank you!